Mathlib/Analysis/Calculus/Deriv/Basic.lean

```diff
@@ -69,8 +69,9 @@ and they more frequently lead to the desired result.
 
 We set up the simplifier so that it can compute the derivative of simple functions. For instance,
 ```lean
-example (x : ℝ) : deriv (fun x ↦ cos (sin x) * exp x) x = (cos(sin(x))-sin(sin(x))*cos(x))*exp(x) :=
-by { simp, ring }
+example (x : ℝ) :
+    deriv (fun x ↦ cos (sin x) * exp x) x = (cos(sin(x))-sin(sin(x))*cos(x))*exp(x) := by
+  simp, ring
```

Comment:
But I'm confident that the comma no longer works
```suggestion
  simp; ring
```

Reply:
Oh, indeed!